bin/compile

```diff
@@ -58,7 +58,7 @@ load_env_vars() {
 }
 load_env_vars "FREETDS_VERSION" "FREETDS_ARCHIVE_NAME" "TDS_VERSION" "FREETDS_REBUILD"
 
-FREETDS_VERSION="${FREETDS_VERSION:-1.00.21}"
+FREETDS_VERSION="${FREETDS_VERSION:-1.00.109}"
```

Comment:
I guess the latest 1.00.x version is now

freetds-1.00.112.tar.gz | 2019-04-29 09:22 | 2.9M |  
-- | -- | -- | --